.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: node_js
+
+node_js:
+  - 10
+
+services:
+  - postgresql
+
+before_script:
+  - psql -c 'create database boss;' -U postgres
+
+install:
+  - npm install
+  - rm secrets-sample.json
```

Comment:
why is this required?

Reply:
because we might be needing working secrets (form oneAuth) and making them public is something we wont like.

So this removes the given secrets-sample and then dumps the data from env variable into the file.

Reply:
Why do you think a committed file in an open source project will contain private secrets?

Reply:
okey I git it, it should be secrets.json